src/wallet/rpc/wallet.cpp

```diff
@@ -234,8 +236,18 @@ static RPCHelpMan loadwallet()
                     }
                 },
                 RPCExamples{
-                    HelpExampleCli("loadwallet", "\"test.dat\"")
-            + HelpExampleRpc("loadwallet", "\"test.dat\"")
+                    "\nLoad wallet with files under wallets/foo/:\n"
+                    + HelpExampleCli("loadwallet", "\"foo\"")
+                    + HelpExampleRpc("loadwallet", "\"foo\"")
+                    + "\nLoad wallet with files under wallets/foo/specialWallet/:\n"
+                    + HelpExampleCli("loadwallet", "\"foo/specialWallet/\"")
+                    + HelpExampleRpc("loadwallet", "\"foo/specialWallet/\"")
+                    + "\nLoad wallet using absolute path:\n"
+                    + HelpExampleCli("loadwallet", fs::PathToString(absolutePath))
+                    + HelpExampleRpc("loadwallet", fs::PathToString(absolutePath))
```

Comment:
nit: Might be good to add quotes around the path like the other examples in case there are spaces in usernames or other part of the directory?